<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.203704 rgba(36, 57, 187, 255), stop:1 rgba(162, 164, 255, 255));}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>911</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 16pt &quot;Space Grotesk SemiBold&quot;;color:rgb(253, 228, 255)
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;Suicide prevention classifier&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="accountLineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 0);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: rgb(253, 228, 255);
font: 12pt &quot;Space Grotesk&quot;;
color: rgb(253, 228, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Space Grotesk Medium&quot;;
color: rgb(253, 228, 255);</string>
    </property>
    <property name="text">
     <string>@</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 12pt &quot;Space Grotesk Medium&quot;;
color: rgb(253, 228, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Add Twitter account identifiers to analyze.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-width: 0px;
border-radius:10px;
border-color: rgb(251, 234, 255);
background-color: rgb(251, 234, 255);
font: 11pt &quot;Space Grotesk&quot;;
color: rgb(0, 0, 127)
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analyzeButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>540</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-width: 0px;
border-radius:10px;
border-color: rgb(251, 234, 255);
background-color: rgb(251, 234, 255);
font: 11pt &quot;Space Grotesk&quot;;
color: rgb(0, 0, 127)
</string>
    </property>
    <property name="text">
     <string>Analyze accounts</string>
    </property>
   </widget>
   <widget class="QListWidget" name="accountListWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>260</y>
      <width>411</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(251, 234, 255);
font: 11pt &quot;Space Grotesk&quot;;
color: rgb(0, 0, 127);
padding:5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delAllButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>460</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-width: 0px;
border-radius:10px;
border-color: rgb(251, 234, 255);
background-color: rgb(36, 57, 187);
font: 11pt &quot;Space Grotesk&quot;;
color: rgb(251, 234, 255);
text-align: center;
</string>
    </property>
    <property name="text">
     <string>Empty list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delAccountButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>430</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-width: 0px;
border-radius:10px;
border-color: rgb(251, 234, 255);
background-color: rgb(34, 57, 187);
font: 11pt &quot;Space Grotesk&quot;;
color: rgb(251, 234, 255);
text-align: center;


</string>
    </property>
    <property name="text">
     <string>Delete account</string>
    </property>
   </widget>
   <widget class="QLabel" name="repetidoLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>490</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 11pt &quot;Space Grotesk SemiBold&quot;;
color:rgb(49, 0, 63);
text-align: center;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="lenguageComboBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(36, 57, 187);
font: 10pt &quot;Space Grotesk&quot;;
color:rgb(244, 216, 255);
border-style: outset;
border-width: 0px;
border-radius: 10px;
border-color: rgb(253, 228, 255);



</string>
    </property>
    <item>
     <property name="text">
      <string>English</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Espa√±ol</string>
     </property>
    </item>
   </widget>
   <widget class="QToolButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 0);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: rgb(253, 228, 255);
font: 12pt &quot;Space Grotesk&quot;;
color: rgb(253, 228, 255);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::LeftArrow</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addTestAccButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>200</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-width: 0px;
border-radius:10px;
border-color: rgb(251, 234, 255);
background-color: rgb(251, 234, 255);
font: 10pt &quot;Space Grotesk&quot;;
color: rgb(0, 0, 127)
</string>
    </property>
    <property name="text">
     <string>Add test accounts</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>640</y>
      <width>591</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar#progressBar{
     background-color: #ffe6ff;

     color: #ffe6ff;
     border-style: outset;
     border-width: 2px;
     border-color: #ffcccc;
     border-radius: 7px;
     text-align: left; 
     font: 10pt &quot;Space Grotesk&quot;; }


 QProgressBar::chunk#progressBar{
        background-color: #3333ff; }</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
    <property name="format">
     <string>   %p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="repetidoLabel_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 11pt &quot;Space Grotesk SemiBold&quot;;
color:rgb(49, 0, 63);
text-align: center;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="updateLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>610</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 11pt &quot;Space Grotesk Medium&quot;;
color: rgb(4, 0, 58);
text-align: right;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;TextLabel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>680</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-width: 0px;
border-radius:10px;
border-color: rgb(251, 234, 255);
background-color: rgb(34, 57, 187);
font: 11pt &quot;Space Grotesk&quot;;
color: rgb(251, 234, 255);
text-align: center;
</string>
    </property>
    <property name="text">
     <string>Stop analyzing</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>815</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

image: url(:/foto/logo2.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="../../../../Desktop/test.qrc">:/foto/lenguahe2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="goToResultsButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>630</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-width: 0px;
border-radius:10px;
border-color: rgb(251, 234, 255);
background-color: rgb(34, 57, 187);
font: 11pt &quot;Space Grotesk&quot;;
color: rgb(251, 234, 255);
text-align: center;</string>
    </property>
    <property name="text">
     <string>Go to results</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../../Downloads/Space_Grotesk/fileLetraFont.qrc"/>
  <include location="../../../../Desktop/test.qrc"/>
 </resources>
 <connections/>
</ui>
